frontend/.happy/terraform/envs/dev/main.tf

```diff
@@ -0,0 +1,28 @@
+# Auto-generated by 'happy infra'. Do not edit
+# Make improvements in happy, so that everyone can benefit.
+module "stack" {
+  source           = "git@github.com:chanzuckerberg/happy//terraform/modules/happy-stack-eks?ref=jgadling/optional-datadog"
```

Comment:
Currently referencing the branch that has changes to make data-dog optional. This is to prevent exposing aws account ids for security compliance.